onnx/defs/tensor/utils.h

```diff
@@ -31,4 +31,12 @@ void resizeShapeInferenceHelper_opset7_to_10(
     const TensorShapeProto& input_shape,
     const std::vector<float>& scales_data,
     TensorShapeProto* output_shape);
+
+static const char* NonZero_ver9_doc = R"DOC(
```

Comment:
I guess this should not be `static`?

Reply:
I suppose it must be static here. Otherwise, it will be defined twice in onnx/defs/tensor/def.cc and onnx/defs/tensor/old.cc and bump into a warning.

Reply:
well, but if the goal is to avoid duplicate copies, we need to move it into one of the .cc files, and leave it as a simple extern declaration in the include file? Otherwise, wouldn't you end up with multiple copies anyway? (In any case, there is a CI failure relating to this, that needs some resolution.)

Reply:
Thanks for catching this! Yes, I believe extern without static should be the right approach to avoid duplicate copies. Will update it later and let you review.